packages/fxa-content-server/app/scripts/views/password_strength/password_strength_balloon.js

```diff
@@ -93,6 +94,7 @@ class PasswordStrengthBalloonView extends BaseView {
 
 Cocktail.mixin(
   PasswordStrengthBalloonView,
+  ExternalLinksMixin,
```

Comment:
Included this because I noticed the strong PW link didn't have `noopener noreferrer` on it.